netlify.toml

```diff
@@ -7,4 +7,4 @@
 [build.environment]
   NPM_FLAGS = "--version"
 [build]
-  command = "npx pnpm install --store=node_modules/.pnpm-store && npx pnpm build"
+  command = "npx pnpm@7 install --store=node_modules/.pnpm-store && npx pnpm@7 build"
```

Comment:
I'm setting an explicit version here so our builds don't fail again on the next breaking change...